readinglist/__init__.py

```diff
@@ -39,4 +41,5 @@ def main(global_config, **settings):
         config.registry.settings['cliquet.paginate_by'] = 100
 
     config.scan("readinglist.views")
-    return config.make_wsgi_app()
+    app = config.make_wsgi_app()
```

Comment:
yes of  course. The plan is to actually get this in cliquet and not in readinglist, though.